src/main/java/sirius/biz/jobs/batch/file/XMLExportJobFactory.java

```diff
@@ -20,16 +20,16 @@
  */
 public abstract class XMLExportJobFactory extends FileExportJobFactory {
 
-    protected final Parameter<Boolean> requireValidFile =
+    protected final BooleanParameter requireValidFileBuilder =
```

Comment:
make a method and return a fresh instance each time? this avoids accidentally modifying a shared object

Reply:
maybe also define the name as reusable constant or a placeholder parameter as was there before.

Reply:
Added a shared name. Builder removed completely as this is handled internally by this class.